src/main/java/com/google/sps/servlets/TopChartsCluster.java

```diff
@@ -0,0 +1,66 @@
+
+package com.google.sps.servlets;
+import java.util.ArrayList;
+
+public class TopChartsCluster {
```

Comment:
Nice job creating a separate class for TopChartsCluster!

I would move these to another package, though, such as `src/main/java/com/google/sps/models`